library/hawkular_alerts_group_member.py

```diff
@@ -222,18 +238,20 @@ def main():
         if module.params[arg] in (None, ''):
             module.fail_json(msg="missing required argument: {}".format(arg))
 
-    hostname    = module.params['hawkular_api_hostname']
-    port        = module.params['hawkular_api_port']
-    token       = module.params['hawkular_api_auth_token']
-    tenant      = module.params['tenant']
-    group_id    = module.params['group_id']
-    member_id   = module.params['id']
-    member_name = module.params['name']
-    data_id_map = module.params['data_id_map']
-    scheme      = module.params['scheme']
-    state       = module.params['state']
-    verify_ssl  = module.params['verify_ssl']
-    ca_file     = module.params['ca_file_path']
+    hostname           = module.params['hawkular_api_hostname']
+    port               = module.params['hawkular_api_port']
+    token              = module.params['hawkular_api_auth_token']
+    tenant             = module.params['tenant']
+    group_id           = module.params['group_id']
+    member_id          = module.params['id']
+    member_name        = module.params['name']
+    member_description = module.params['description']
```

Comment:
I would drop the `member` prefix from both `member_description` and `member_tags`
(I know it `member_name` has this prefix as well, also unneeded)